.travis.yml

```diff
@@ -1,12 +1,77 @@
 language: cpp
 dist: trusty
 
-# In order to catch as many weird edge cases in the code with -Werror as
-# possible, we want to test a large range of old and new compilers on both
-# Linux and macOS. This gives us the best possible coverage while maintaining
-# compatibility with a large number of compilers.
-matrix:
+stages:
+    - c-cpp
+    - python
+
+env:
+    global:
+        # Uploading wheels to PyPI, TWINE_PASSWORD is set to a PyPI API token in Travis settings
+        - TWINE_USERNAME=__token__
+
+.testing: &testing
```

Comment:
Anchor should also be renamed to c-cpp or ccpp if c-cpp can't pass as identifier.

Reply:
Done